client/src/components/Login.vue

```diff
@@ -102,6 +102,15 @@ export default {
             formValues[inputs[i].id] = inputs[i].value
           }
         }
+        restFactory.postVerifySlackEmail(formValues).then((res) => {
+          if (res.status === 200) {
+            console.log(res.data.data)
```

Comment:
Console log? Are you planning on changing this in part 2?

Reply:
Yes, this will be changed. For now it's just there to validate that we are hitting the endpoint.